pcdsdevices/interface.py

```diff
@@ -383,7 +383,7 @@ def get_value(signal):
 
 
 def get_units(signal):
-    attrs = ('units', 'egu', 'derived_units')
+    attrs = ('derived_units', 'units', 'egu')
```

Comment:
This is just a workaround of an issue I was having with the `units` ... i hope it does not break anything else... but I can switch this back.

The problem is that when I get the `lxt.status_info()` - the `units` are in `ns` - which is wrong - what we want is the `derived_units` (`s` in my case).

```
{'name': 'lxt',
 'kind': <Kind.config|normal: 3>,
 'is_device': True,
 'position': -5.597463492824768e-06,
 'setpoint': {'name': 'lxt_setpoint',
  'kind': <Kind.hinted: 5>,
  'is_device': False,
  'value': -5.597463492824768e-06,
  'units': 'ns'},
 'notepad_setpoint': {'name': 'lxt_notepad_setpoint',
  '.......
```

By changing the `attrs = (derived_units, ...)` to have the `derived_units` first element in the tuple - it seems like in the `status_info()` it picks the `derived_units` first and returns that:

```
{'name': 'lxt',
 'kind': <Kind.config|normal: 3>,
 'is_device': True,
 'position': -5.597463492824768e-06,
 'setpoint': {'name': 'lxt_setpoint',
  'kind': <Kind.hinted: 5>,
  'is_device': False,
  'value': -5.597463492824768e-06,
  'units': 's'},
 '.......
```

but as @ZryletTC pointed out to me in the slack chat - the  `UnitConversionDerivedSignal` most likely needs to be fixed, since the metadata for `lxt.setpoint.metadata` still has `units` as `ns` (like the original units)

I think we need to open a new PR for this issue rather than fixing it in this one - i personally don't know yet what needs fixing...

Reply:
A new issue has been added for this https://github.com/pcdshub/pcdsdevices/issues/767

Reply:
bizarre workaround, but if it works, it works!